-- Use the version of setuptools available in FreeBSD ports instead of requiring setuptools>=77.
-- Also remove SPDX license fields not supported by the available setuptools version.

--- pyproject.toml.orig	2026-10-11 17:34:52 UTC
+++ pyproject.toml
@@ -1,12 +1,10 @@
 [build-system]
-requires = ["setuptools>=77"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "caio"
 version = "0.12.9"
-license = "Apache-2.0"
-license-files = ["COPYING"]
 description = "Asynchronous file IO for Linux MacOS or Windows."
 readme = "README.md"
 authors = [{ name = "Dmitry Orlov", email = "me@mosquito.su"}]
